mscore/editelement.cpp

```diff
@@ -60,8 +60,8 @@ void ScoreView::updateGrips()
 
             // updateGrips returns grips in page coordinates,
             // transform to view coordinates:
-
-            const QPointF pageOffset(editData.element->findAncestor(ElementType::PAGE)->pos());
+            Element* el = editData.element->findAncestor(ElementType::PAGE);
```

Comment:
I suggest renaming `el` to `page` and making it `const`:

`Element* const page = editData.element->findAncestor(ElementType::PAGE);`

Reply:
dito